extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -357,7 +360,6 @@ private function getContributorsDetails( $contributors, $avatarSize = AvatarServ
 			$user = User::newFromId( $contributor[ 'userId' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "userId" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)